### DISCUSSION TITLE
走进 阿里/蚂蚁的 OSPO

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#17] [日常讨论]
3 月 3 日 至 4 日，开源社 & X-lab 的部分成员小伙伴们，一起走访了阿里、蚂蚁的开源办公室，充分感受到了互联网企业在建设 OSPO 方面的力量以及在开源社区的投入，也欢迎大家能够更加广泛的进行交流，助力全球开源生态的美好建设~

![ali-ant](https://user-images.githubusercontent.com/15010826/157354452-27602ce5-f4f6-4a99-ac0f-ff10cf751400.jpg)

**Comment by wangleo61** [comment]
蚂蚁推动开源的值得尊敬，期待期待

**Comment by will-ww** [comment]
又是一年的 3 月份~

![image](https://user-images.githubusercontent.com/15010826/225886604-93dfeb61-5971-4e8c-8ee7-9b2f46e9e1df.png)
